src/Services/Masa.Auth.Service.Admin/Application/Subjects/QueryHandler.cs

```diff
@@ -388,12 +388,16 @@ public async Task GetThirdPartyUsersAsync(ThirdPartyUsersQuery query)
         if (query.EndTime is not null)
             condition = condition.And(tpu => tpu.CreationTime <= query.EndTime);
 
-        if (query.UserId != Guid.Empty)
-            condition = condition.And(tpu => tpu.Id == query.UserId);
+        if (query.ThirdPartyId is not null)
```

Comment:
`condition.And(query.ThirdPartyId is not null, tpu => tpu.ThirdPartyIdpId == query.ThirdPartyId)`